elifetools/utils_html.py

```diff
@@ -106,6 +108,23 @@ def replace_xref_tags(s):
 
     return s
 
+def replace_related_object_tags(s):
+    for tag_match in re.finditer("<(related-object.*?)>", s):
+        xlink_match = re.finditer('xlink:href="(.*)"', tag_match.group())
+        if xlink_match:
+            try:
+                xlink = next(xlink_match).group(1)
+                if not xlink[0:4] in ['http', 'ftp:']:
+                    xlink = 'http://' + xlink
```

Comment:
can you add an example of a non-http and non-ftp xlink here? just a simple:

```python
# for cases like 'foo.bar/baz'
xlink = 'http://' + xlink
```

Reply:
Agreed, I was a little confused when I first saw it (again). This code is duplicated now too, which I think deserves a refactor before I merge it.